### ISSUE TITLE
Encoding issues with folder names containing german umlaut

### ISSUE THREAD

**Issue by bentolor** [joeyates/imap-backup#80]
Hi @joeyates ! Thanks for you helpful tool. Especially the `setup` functionality is a joy of simplicity. Love it.

A little issue i stumbled upon: `imap-backup status` lists the following:

```
Archiv: 0
Entw&APw-rfe: 0
Gel&APY-scht: 135
Gesendet: 31
INBOX: 1799
OUTBOX: 0
Spamverdacht: 115
```

The proper foldernames are _Entwürfe_ and _Gelöscht_; Not sure about the background of the `&APw-` for ü.

Fortunately it seems that the backups seems to work nonetheless, though…

**Comment by joeyates** [comment]
Hi @bentolor ,

Thanks for the report. Glad it works nonetheless. I'll leave this open till we get a fix.

**Comment by igorkulman** [comment]
Same problem with central European characters.

**Comment by RemyFV** [comment]
same for every accent used in French, too.

```
&AMk-l&AOk-ments supprim&AOk-s_old.imap
&AMk-l&AOk-ments supprim&AOk-s_old.mbox
```
 great tool otherwise :)

**Comment by joeyates** [comment]
Released as v3.1.0
